probables/blooms/bloom.py

```diff
@@ -13,7 +13,7 @@
 from ..exceptions import InitializationError, NotSupportedError
 from ..hashes import HashFuncT, HashResultsT
 from ..utilities import is_hex_string, is_valid_file
-from .basebloom import BaseBloom
+from .basebloom import BaseBloom, HashFuncT
```

Comment:
Not sure why `HashFuncT` is imported here as it is pulled in from the `hashes` utility file.

Reply:
Fixed.